templates/blog/post_detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <div class="post">
+    <h2 class="text-center">{{ post.title }}</h2>
+    <hr style="height: 2px; background-color: #9D7753">
+    <p style="font-size: large">{{ post.content }}</p>
```

Comment:
Inline CSS is generally considered bad practice due to issues with maintenance and scalability. It's recommended to move this to an external CSS file.